Code/Engine/Foundation/Reflection/Implementation/RTTI.h

```diff
@@ -204,6 +204,8 @@ class EZ_FOUNDATION_DLL ezRTTI : public ezEnumerable<ezRTTI>
 /// \brief The interface for an allocator that creates instances of reflected types.
 struct EZ_FOUNDATION_DLL ezRTTIAllocator
 {
+  virtual ~ezRTTIAllocator();
```

Comment:
@C-Core your resent scripting additions now use a ezUniquePtr<ezRTTIAllocator> which lead to a call to a non virtual destructor ;-)